hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/metrics2/impl/TestMetricsSystemImpl.java

```diff
@@ -639,4 +639,23 @@ public Boolean get() {
   private static String getPluginUrlsAsString() {
     return "file:metrics2-test-plugin.jar";
   }
+
+  @Test
+  public void testMetricSystemRestart() {
+    MetricsSystemImpl ms = new MetricsSystemImpl("msRestartTestSystem");
+    TestSink ts = new TestSink();
+    String sinkName = "restartTestSink";
+
+    try {
+      ms.start();
+      ms.register(sinkName, "", ts);
+      assertNotNull("an adapter should exist for each sink", ms.getSinkAdapter(sinkName));
```

Comment:
we're generally still 80 chars wide, I'm afraid...put the actual probe on a new line.
nice to see some text for the assertion tnough -appreciated

Reply:
Fixed it, please review (added more info to the message)